libs/ledgerjs/packages/hw-app-btc/src/BtcOld.ts

```diff
@@ -7,6 +7,7 @@ import { createTransaction } from "./createTransaction";
 import type { AddressFormat } from "./getWalletPublicKey";
 import { getWalletPublicKey } from "./getWalletPublicKey";
 import { pathArrayToString, pathStringToArray } from "./bip32";
+import { signMessage } from "./signMessage";
```

Comment:
Thanks for the feedback @haammar-ledger.

Although appending `Old` would make it more consistent with the `BtcOld` class, it would become inconsistent with the other old functions:

- `createTransaction` from `createTransaction.ts`
- `getWalletPublicKey` from `getWalletPublicKey.ts`

So if you don't mind, I'd keep it as is to not change too many otherwise untouched files with this PR.